hangman.py

```diff
@@ -0,0 +1,191 @@
+import os
+
+HANGMAN_ASCII_ART = ("""welcome to the game hangman /
+  _    _                                         
+ | |  | |                                        
+ | |__| | __ _ _ __   __ _ _ __ ___   __ _ _ __  
+ |  __  |/ _` | '_ \ / _` | '_ ` _ \ / _` | '_ \ 
+ | |  | | (_| | | | | (_| | | | | | | (_| | | | |
+ |_|  |_|\__,_|_| |_|\__, |_| |_| |_|\__,_|_| |_|
+                      __/ |                      
+                     |___/""")
+
+MAX_TRIES = (6)
+
+HANGMAN_PHOTOS = {"stage 0": "x-------x", "stage 1": """x-------x
+|
+|
+|
+|
+|""", "stage 2": """x-------x
+|       |
+|       0
+|
+|
+|""", "stage 3": """x-------x
+|       |
+|       0
+|       |
+|
+|
+""", "stage 4": """x-------x
+|       |
+|       0
+|      /|\\
+|
+|""", "stage 5": """x-------x
+|       |
+|       0
+|      /|\\
+|      /
+|""", "stage 6": """x-------x
+|       |
+|       0
+|      /|\\
+|      / \\
+|"""}
+
+def opening_screen():
+    """
+    the func prints the opening screen
+    :return void:
+    """
+    print(HANGMAN_ASCII_ART, "Your number of max tries is: {}".format(MAX_TRIES), sep='\n')
+
+def fix_path(file_path):
+    """
+    the funcs rempve
+    :param file_path:
+    :return:
+    """
+    if (file_path[0] == "\"" or file_path[0] == "\'") and (file_path[-1] == "\"" or file_path[-1] == "\'"):
+        return file_path[1:-1]
+    else:
+        return file_path
+
+def get_data_from_user():
+    """
+    the funcs get the path file and the index of word from the user
+    :return file path - the path of file - string, the index of the word - int:
+    """
+    file_path_of_words = fix_path(input("Enter file path: "))
+    while not os.path.exists(file_path_of_words):
+        print("The path is not valid or doesn't point to a file.")
+        file_path_of_words = fix_path(input("Enter file path: "))
+    index_of_word = input("Enter_index: ")
```

Comment:
`word_index` is better also, what are we indexing

Reply:
changed, the secret word